engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -97,19 +100,15 @@ case class OldStyleSingleWorkflowRunnerActor(source: WorkflowSourceFiles,
   }
 
   when (RequestingOutputs) {
-    case Event(WorkflowManagerWorkflowOutputsSuccess(id, outputs), data) =>
-      // Outputs go to stdout
+    case Event(RequestComplete((StatusCodes.OK, outputs: JsObject)), _) =>
```

Comment:
I think we should be careful about making API depend on engine, and not vice versa. Unless we consider `SingleRunnerActor` an API-package actor?